attributes/default.rb

```diff
@@ -0,0 +1,48 @@
+default['qubit_bamboo']['home'] = '/opt/bamboo'
+default['qubit_bamboo']['version'] = '0.2.12'
+
+default['qubit_bamboo']['config_path'] = "#{node['qubit_bamboo']['home']}/production.json"
+
+default['qubit_bamboo']['log_path'] = '/var/log/bamboo-server.log'
+
+default['qubit_bamboo']['config']['marathon_port'] = 8080
+default['qubit_bamboo']['config']['marathon_hosts'] = ["http://localhost:#{node['qubit_bamboo']['config']['marathon_port']}"]
+
+default['qubit_bamboo']['config']['port'] = 8000
+
+default['qubit_bamboo']['config']['zk']['hosts'] = ['localhost']
+default['qubit_bamboo']['config']['zk']['path'] = ''
+default['qubit_bamboo']['config']['zk']['reporting_delay'] = 5
+
+default['qubit_bamboo']['config']['haptoxy']['template_path'] = "#{node['qubit_bamboo']['home']}/haproxy_template.cfg"
+default['qubit_bamboo']['config']['haproxy']['output_path'] = '/etc/haproxy/haproxy.cfg'
+default['qubit_bamboo']['config']['haproxy']['ReloadCommand'] = 'haproxy -f /etc/haproxy/haproxy.cfg -p /var/run/haproxy.pid -D -sf $(cat /var/run/haproxy.pid)'
+
+default['qubit_bamboo']['config']['statsd']['enabled'] = false
+default['qubit_bamboo']['config']['statsd']['host'] = 'localhost'
+default['qubit_bamboo']['config']['statsd']['port'] = 8125
+default['qubit_bamboo']['config']['statsd']['prefix'] = 'bamboo'
+
+default['qubit_bamboo']['config'] = {
```

Comment:
OPT: IMO, all options set under this hash do not need to be set somewhere else before, as it is quite easy to overwrite attributes as needed.